staging/src/k8s.io/apiserver/pkg/util/logs/logs.go

```diff
@@ -22,6 +22,7 @@ import (
 	"log"
 	"time"
 
+	"github.com/coreos/etcd/clientv3"
```

Comment:
Is clientv3 the appropriate place for this?  Wouldn't the storage backend init() be the more appropriate place?

Reply:
Good question. Here we do global logging configuration, so side effect are intended. In the storage backend init I would rather not expect side-effects.